File: ZKUIComponents/src/main/java/com.zkteam.ui.components/activity/EmptyActivity.java
package com.zkteam.ui.components.activity;

import android.content.Intent;
import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;
import com.zkteam.ui.components.R;

/**
 * Created by renxuelong on 2019-07-10
 */
public class EmptyActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_empty);

        findViewById(R.id.btn_setting).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(EmptyActivity.this, SettingActivity.class));
            }
        });
    }
}

File: ZKUIComponents/src/main/java/com.zkteam.ui.components/activity/EmptyActivity.kt
package com.zkteam.ui.components.activity

import android.content.Intent
import android.os.Bundle
import android.view.View
import com.zkteam.sdk.base.ZKBaseActivity
import com.zkteam.ui.components.R
import kotlinx.android.synthetic.main.activity_empty.*

class EmptyActivity: ZKBaseActivity() {
    override fun getLayoutId(): Int {
        return R.layout.activity_empty
    }

    override fun initData(bundle: Bundle?) {
        //function
    }

    override fun initLifecycleObserve() {
        //function
    }

    override fun initListener() {
        btn_setting.setOnClickListener {
            startActivity(Intent(this, SettingActivity::class.java))
        }
    }

    override fun initViews(contentView: View) {
        //function
    }

    override fun onDebouncingClick(view: View) {
        //function
    }
}